internal/app/handle.go

```diff
@@ -128,3 +170,41 @@ func handleGokuDefault(w http.ResponseWriter, r *http.Request) {
 	w.Write(d)
 	logger.Info().Msg("Message written\n")
 }
+
+// the elasticClient function creates an elastic search client and
+// adds an index named fighters that will use the mapping variable to set
+// the layout of its body if it doesn't already exist.
+func elasticClient() (context.Context, *elastic.Client) {
+	logger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: time.RFC3339}).With().Caller().Timestamp().Logger()
```

Comment:
i see you instantiating this logger a lot. we can declare the variable outside the scope of these functions, and initialize it as part of an init() function. i'll show you how to do this before scrum :+1: